registry.ts

```diff
@@ -42,14 +42,14 @@ export function defaultName(that: RegistryUrl): string {
   return n[1];
 }
 
-async function githubDownloadRelases(
+async function githubDownloadReleases(
   owner: string,
   repo: string,
   lastVersion: string | undefined = undefined,
 ): Promise<string[]> {
   let url = `https://github.com/${owner}/${repo}/releases.atom`;
   if (lastVersion) {
-    url += `?${lastVersion}`;
+    url += `?after=${lastVersion}`;
```

Comment:
does this actually work? I tested `https://github.com/denoland/deno/releases.atom?after=v1.0.1`

and it doesn't show 1.0.2... I guess I haven't checked out this branch to verify...

Reply:
It's a weird choice of name for the query param, the meaning is opposite what you'd think. Since the feed is from newest to oldest, `?after=v1.0.1` means "show me releases **before** v1.0.1".

Using your link as an example, if you go to `https://github.com/denoland/deno/releases.atom?after=v1.0.1` you will see v1.0.0 followed by all the 1.0.0 release candidates